google-cloud-identity_toolkit-v2/.repo-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-identitytoolkit-v2/latest",
+    "distribution_name": "google-cloud-identitytoolkit-v2",
+    "language": "ruby",
+    "name_pretty": "Google Cloud Identitytoolkit V2 API",
```

Comment:
The formatting of "Identitytoolkit" seems incorrect for a pretty name. It should be something like:

```
"name_pretty": "BigQuery Data Transfer Service V1 API"
```